docs/use/Interact-with-dApps/index.md

```diff
@@ -3,18 +3,44 @@ sidebar_position: 4
 slug: '/dapps'
 ---
 
-# Interact with dApps
+# Use dApps
 
-Before engaging with our dApps, it is important to have a wallet. If you have not installed a wallet, please visit our [wallet's](../wallet) section to select an appropriate wallet.
+Before engaging with [dApps](../intro#what-are-dapps), it is important to have a wallet with some Evmos tokens. If you have not installed a wallet, please visit our [wallet's](../wallet) section to select an appropriate wallet. If you do not have any EVMOS tokens, you can acquire them by reading the [Onramp to Evmos](./../transfer#onramp-evmos) strategies.
```

Comment:
The links for dapps and wallet are not working.

Reply:
Can we fix links at the end of the refactor? So many pages are changing, so handling with the links is taking a lot of time

Reply:
Sure but my latest PR should address most if not all of the current issue. So if we merge that PR, then hopefully, these entities should not be an issue unless its truly broken.